src/main/config/config.c

```diff
@@ -454,6 +454,8 @@ static void resetConf(void)
     masterConfig.gyro_sync_denom = 4;
 #endif
     masterConfig.gyro_soft_lpf_hz = 100;
```

Comment:
In config.c or in the filters?

For filters something like this would look nice:

```
typedef struct biquadFilterConfig_s {
    float filterFreq;
    uint32_t refreshRate;
    float Q;
    filterType_e filterType;
} biquadFilterConfig_t;
void biquadFilterInit(biquadFilter_t *filter, biquadFilterConfig_t *filterConfig);
```